### PR TITLE
Use common pkger

### PR DESCRIPTION
**Pull request by bastien-gatling**

### CODE REVIEW

File: src/main/groovy/io/gatling/gradle/GatlingEnterprisePackageTask.groovy [hunk 1]
```diff
@@ -1,79 +1,88 @@
...
+            basePluginExtension.libsDirectory.get().asFile.getAbsolutePath(),
+            project.name +
+                jarVersion +
+                '-' + ARCHIVE_CLASSIFIER +
+                JAR_EXTENSION)
+    }
+
+    private List<File> getClassDirectories() {
+        project.getAllprojects().collect { p ->
+            JavaPluginExtension javaPluginExtension = p.getExtensions().getByType(JavaPluginExtension.class)
```
Comment (slandelle) [inline src/main/groovy/io/gatling/gradle/GatlingEnterprisePackageTask.groovy:81]: What if this project uses Scala or Kotlin?

Comment (bastien-gatling) [reply src/main/groovy/io/gatling/gradle/GatlingEnterprisePackageTask.groovy:72]: It's seems to be the plugin used by all the jvm related project.

As they themselve say in [the documentation](https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/JavaPluginExtension.html): 
> This extension is added by the JavaBasePlugin and would be more appropriately named the JvmPluginExtension extension. It is used to configure many of the project's JVM-related settings and behavior.

I've just tested it with the gatling-gradle-plugin-demo-kotlin project.

File: src/main/groovy/io/gatling/gradle/GatlingEnterprisePackageTask.groovy [hunk 2]
```diff
@@ -1,79 +1,88 @@
...
-                logger.debug("Detected Gatling compile version: {}", id.version)
-                return id.version
+
+    private File getJarFile() {
+        String jarVersion = skipVersionIfUnspecified((String) project.version)
+        new File(
+            basePluginExtension.libsDirectory.get().asFile.getAbsolutePath(),
+            project.name +
+                jarVersion +
+                '-' + ARCHIVE_CLASSIFIER +
```
[the remark this answers is not in the document]
Comment (bastien-gatling) [reply src/main/groovy/io/gatling/gradle/GatlingEnterprisePackageTask.groovy:66]: Actually, that's my intention. I started to map what you done in the maven plugin (with the .shaded.jar extension) and the integration tests were no longer passing.

So I reproduce the name schema of the [AbstractArchiveTask](https://github.com/gradle/gradle/blob/4bce19d8dbe01a25cc7eebd960b3f940d8ae7b4b/subprojects/core/src/main/java/org/gradle/api/tasks/bundling/AbstractArchiveTask.java#L72) that the previous task inherit before.

Now, IT still ok, and there is the version before de tests classifier when a version is defined.

**Review by slandelle — APPROVED** [review]

**Review by guilgaly — APPROVED** [review]

**Comment by slandelle** [comment]
@bastien-gatling Please rebase. Merge then?

**Pull request merged**